build.gradle.kts

```diff
@@ -29,17 +29,17 @@ repositories {
 dependencies {
 
     val springdocVersion = "1.6.13"
-    val sentryVersion = "6.8.0"
+    val sentryVersion = "6.6.0"
     val navFellesVersion = "1.20220901103347_4819e55"
     val eksterneKontrakterBisysVersion = "2.0_20220609214258_f30c3ce"
     val fellesKontrakterVersion = "2.0_20221121103857_87ffed0"
     val familieKontrakterSaksstatistikkVersion = "2.0_20220216121145_5a268ac"
     val familieKontrakterStønadsstatistikkKsVersion = "2.0_20221123121611_ca933bd"
     val familieKontrakterSkatteetatenVersion = "2.0_20210920094114_9c74239"
-    val tokenValidationSpringVersion = "2.1.8"
+    val tokenValidationSpringVersion = "2.1.7"
     val navFoedselsnummerVersion = "1.0-SNAPSHOT.6"
     val prosesseringVersion = "1.20221110194901_e9e0d90"
-    val restAssuredVersion = "5.3.0"
+    val restAssuredVersion = "5.2.0"
```

Comment:
Er det meningen at mange av avhengighetene skal downgrades?

Reply:
Haha nei, denne slang seg med for noen grunner.